parser/parser.y

```diff
@@ -1823,7 +1826,40 @@ OnDuplicateKeyUpdate:
 		$$ = $5
 	}
 
-/***********************************Insert Statments END************************************/
+/************************************************************************************
+ *
+ *  Replace Statments
+ *
+ *  TODO: support PARTITION
```

Comment:
Add a reference here. SQL statement syntax maybe different for different mysql version. So we need add reference.

We use 5.7 as standard. https://dev.mysql.com/doc/refman/5.7/en/replace.html